filters/f_auto_filters.c

```diff
@@ -73,6 +73,13 @@ static void deint_process(struct mp_filter *f)
         return;
     }
 
+    char* field_parity;
+    switch(opts->field_parity){
```

Comment:
missing a space here, `switch`/`case` should have the same indentation level and `break;` should be on a newline